tutorials/introductory/usage.py

```diff
@@ -141,101 +137,84 @@
 #
 # and to convert a `np.matrix` ::
 #
-#   b = np.matrix([[1,2],[3,4]])
+#   b = np.matrix([[1, 2], [3, 4]])
 #   b_asarray = np.asarray(b)
 #
-# .. _pylab:
-#
-# Matplotlib, pyplot and pylab: how are they related?
-# ====================================================
+# .. _coding_styles:
 #
-# Matplotlib is the whole package and :mod:`matplotlib.pyplot` is a module in
-# Matplotlib.
+# Coding Styles
```

Comment:
Is "Coding Styles" the right phrase here? I find this more mysterious than helpful.  "Object-Oriented and Pyplot APIs?" "...Interfaces"?

Reply:
sure, edited